bluebottle/utils/views.py

```diff
@@ -244,13 +246,9 @@ def get_object(self):
 
     def get(self, request, *args, **kwargs):
         field = getattr(self.get_object(), self.field)
-        response = HttpResponse()
-        response['X-Accel-Redirect'] = field.url
```

Comment:
Not sure if removing this won't break file serving on testing/prod servers.
Do you know @eodolphi ?

Reply:
I re-added it, because tests were relying on it too, so it's probably of some good use ;-)